gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -13684,6 +13684,12 @@ void GUIScript::AssignViewAttributes(PyObject* obj, View* view) const
 	static PyObject* windowClass = PyDict_GetItemString(pGUIClasses, "GWindow");
 	
 	PyObject_SetAttrString(obj, "Flags", DecRef(PyLong_FromLong, view->Flags()));
+	Window* win = view->GetWindow();
+	if (win) {
```

Comment:
Shouldn't we add the field either way, so we can test for None instead of presence on the python side. Or is this handled elsewhere?

Reply:
thats a good question and I should test that. I suspect that it will automatically be `None`, since its defined in `__slots__`, but i'm not sure.